nullaway/src/main/java/com/uber/nullaway/GenericsChecks.java

```diff
@@ -0,0 +1,95 @@
+package com.uber.nullaway;
+
+import com.google.errorprone.VisitorState;
+import com.google.errorprone.util.ASTHelpers;
+import com.sun.source.tree.AnnotatedTypeTree;
+import com.sun.source.tree.AnnotationTree;
+import com.sun.source.tree.ParameterizedTypeTree;
+import com.sun.source.tree.Tree;
+import com.sun.tools.javac.code.Attribute;
+import com.sun.tools.javac.code.Type;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/** Methods for performing checks related to generic types and nullability. */
+public class GenericsChecks {
```

Comment:
For a utility method's class: `public final class [...]`, to prevent instantiation via subtyping.

Reply:
updated it.